src/modals/UpdateReserve/index.tsx

```diff
@@ -0,0 +1,160 @@
+  const { frozen } = balance;
+  const { openHelp } = useHelp();
+
+  // check account non-staking locks
+  const locks = getLocks(activeAccount);
+  const locksStaking = locks.find(({ id }) => id === 'staking');
+  const lockStakingAmount = locksStaking
+    ? locksStaking.amount
+    : new BigNumber(0);
+  // available balance data
+  const fundsFree = planckToUnit(
+    allTransferOptions.freeBalance.minus(frozen.minus(lockStakingAmount)),
+    units
+  );
+
+  const warnings = [];
+  if (fundsFree.isZero()) {
+    warnings.push(<Warning text={t('reserveLimit')} />);
+  }
+  const ReserveUpdated = !(
+    planckToUnit(reserve, units).toString() ===
+    localStorage.getItem(`${network.name}_${activeAccount}_reserve`)
+  );
+
+  const ReserveFeedback = (() => {
+    if (!ReserveUpdated) {
+      return undefined;
+    }
+    return {
+      text: t('updated'),
+      label: 'neutral',
+    };
+  })();
+
+  const sliderProps = {
+    trackStyle: {
+      backgroundColor: 'var(--network-color-primary)',
+    },
+    handleStyle: {
+      backgroundColor: 'var(--background-primary)',
+      borderColor: 'var(--network-color-primary)',
+      opacity: 1,
+    },
+  };
+
+  useEffect(() => {
+    setResize();
+  }, [reserve]);
+
+  return (
+    <>
+      <Close />
+      <ModalPadding>
+        <h2 className="title unbounded" style={{ margin: '0.5rem' }}>
+          {t('updateReserve')}
+        </h2>
+        {warnings.length ? (
+          <ModalWarnings>
+            {warnings.map((warning, index) => (
+              <React.Fragment key={`warning_${index}`}>
+                {warning}
+              </React.Fragment>
+            ))}
+          </ModalWarnings>
+        ) : null}
+        <ActionItem
+          text={t('updateReserve')}
+          inlineButton={
+            <ButtonHelp onClick={() => openHelp('Reserve Balance')} />
+          }
+        />
+
+        <WithSliderWrapper>
+          <h5 className="neutral">
+            {t('reserve')}
+            {ReserveFeedback && (
+              <span className="neutral">{ReserveFeedback.text}</span>
+            )}
+          </h5>
+          <div>
+            <h4 className="current">
+              {planckToUnit(reserve, units).toString()}{' '}
+            </h4>
+            <div className="slider">
+              <Slider
+                max={['polkadot', 'westend'].includes(network.name) ? 3 : 1}
+                value={planckToUnit(reserve, units).toNumber()}
+                step={0.01}
+                onChange={(val) => {
+                  if (typeof val === 'number') {
+                    setReserve(
+                      new BigNumber(unitToPlanck(val.toString(), units))
+                    );
+                  }
+                }}
+                {...sliderProps}
+              />
+            </div>
+          </div>
+          <p>{t('reserveText')}</p>
+          <div className="confirm">
+            <ButtonPrimaryInvert
+              text={t('confirm')}
```

Comment:
Change to `Done`. Confirm suggests that a change is being made when you click the button.

Also add a `Reset` button somewhere to set the reserve amount to zero.